network/src/peer_manager/mod.rs

```diff
@@ -354,6 +362,11 @@ where
             channel_size,
             max_frame_size,
             inbound_connection_limit,
+            // FIXME: Don't make this quota hard coded, also it doesn't support more than max u32 here
+            // Allow 1MB per second
+            inbound_rate_limiter: Arc::new(governor::RateLimiter::keyed(Quota::per_second(
+                NonZeroU32::new(1_000_000).unwrap(),
```

Comment:
also 1 MB/s is a looot we probably can suffice 50 KB/s or substantially less...

Reply:
Yeah, it was an arbitrary limit, and I was pretty sure this was in draft...

Reply:
Done.